qa-tests-backend/src/test/groovy/LocalQaPropsTest.groovy

```diff
@@ -12,7 +12,7 @@ import spock.lang.Specification
 // few predefined expectations, but unfortunately the values drift and that test does not run.
 // these predefined values are probably out of date & the test doesn't verify any functionality
 // and hence skipped
-@IgnoreIf({ Env.REMOTE_CLUSTER_ARCH == "ppc64le" || Env.REMOTE_CLUSTER_ARCH == "s390x" })
+// @IgnoreIf({ Env.REMOTE_CLUSTER_ARCH == "ppc64le" || Env.REMOTE_CLUSTER_ARCH == "s390x" })
```

Comment:
This test is anyways skipped on upstream CI even for X. For P/Z we don't need it either be it on upstream CI or downstream CI.
Uncomment & let this test be ignored on P/Z.